pkg/operator/certrotation/cabundle.go

```diff
@@ -28,6 +28,8 @@ type CABundleConfigMap struct {
 	Namespace string
 	// Name is the name of the ConfigMap to maintain.
 	Name string
+	// Owner is an optional reference to add to the secret that this rotator creates.
+	Owner *metav1.OwnerReference
```

Comment:
Can you please elaborate on how this would be useful?

Reply:
In the HyperShift case, secrets and configmaps have their ownerref used for an index, and event handlers for those GVRs use the owner ref to know what related HostedControlPlane object to reconcile when dependent resources change.